lib/src/ruby/module.cc

```diff
@@ -4,6 +4,7 @@
 #include <internal/ruby/confine.hpp>
 #include <internal/ruby/simple_resolution.hpp>
 #include <facter/facts/collection.hpp>
+#include <facter/logging/logging.hpp>
```

Comment:
Ah, is that the secret sauce here? I'll try sedding that and see if that gets it all working